### PR TITLE
Three seconds is better

### PR DESCRIPTION
**Pull request by Viktor-Yegay**
Studies have shown that 3 seconds is a far better LED delay than 1 second.

http://studies.example.com/optimal-led-delays.html

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(3000);               // wait for a second
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
-  delay(2000);               // wait for a second
+  delay(3000);               // wait for a second
```
[the remark this answers is not in the document]
Comment (Viktor-Yegay-It) [reply blink.ino:23]: If you make that change, I'll be happy to merge this.

[the remark this answers is not in the document]
Comment (Viktor-Yegay-It) [reply blink.ino:23]: xxxx

**Comment by Viktor-Yegay-It** [comment]
ddddd

**Comment by Viktor-Yegay** [comment]
I change it to 4 seconds and also removed some trailing whitespace that I found. Anything else you would like me to do?

**Pull request merged**